scripts/install

```diff
@@ -158,61 +162,13 @@ cp --archive ../conf/plugins.local.php            $final_path/conf
 cp --archive ../conf/plugins.local.php            $final_path/conf/plugins.local.php.bak
 
 #=================================================
-# STORE THE CHECKSUM OF THE CONFIG FILE
-#=================================================
-
-# Calculate and store the config file checksum into the app settings
-#ynh_store_file_checksum --file="$final_path/conf/local.protected.php"
-### Files '$final_path/conf/local.php' and '$final_path/conf/acl.auth.php' can be modified by user, no need to store checksum as they cannot be overwritten safely by the upgrade script
-
-# #=================================================
-# # GENERIC FINALIZATION
-# #=================================================
-# # SECURE FILES AND DIRECTORIES
-# #=================================================
-
-# # Try to use "least privilege" to grant minimal access
```

Comment:
Was "overkill" at the time and thi code is much easier.

```bash
chmod 750 "$datadir"
chmod -R o-rwx "$datadir"
chown -R $app:www-data "$datadir"
```

https://github.com/YunoHost/example_ynh/blob/master/scripts/install#L225-L227

Is there "good practices" from other apps or in the "packaging doc" for managing "linux user rights" ? Otherwize, it will stay like this

Reply:
The part I removed, it's just comments.

Since YunoHost 4.2.2, with the merge of https://github.com/yunohost/yunohost/pull/1188, the good practice is to create a system user for the app, give this user the rights on the folders it needs, execute the install steps as the user and execute the systemd service as the user, it minimize security issues

Reply:
> The part I removed, it's just comments.

My bad... 



> Since YunoHost 4.2.2, with the merge of [YunoHost/yunohost#1188](https://github.com/YunoHost/yunohost/pull/1188), the good practice is to create a system user for the app, give this user the rights on the folders it needs, execute the install steps as the user and execute the systemd service as the user, it minimize security issues

Thanks for this!